polyA/calculate_score.py

```diff
@@ -30,11 +32,11 @@ def calculate_score(
     alignment score
 
     >>> sub_mat = {"AA":1, "AT":-1, "TA":-1, "TT":1}
-    >>> calculate_score(-5, -25, "AT", "AT", "", "", sub_mat)
+    >>> calculate_score(-5, -25, "AT", "AT", "", "", sub_mat, None)
     2.0
-    >>> calculate_score(-5, -25, "-T", "AT", "A", "A", sub_mat)
+    >>> calculate_score(-5, -25, "-T", "AT", "A", "A", sub_mat, None)
     -24.0
-    >>> calculate_score(-5, -25, "-T", "AT", "-", "", sub_mat)
+    >>> calculate_score(-5, -25, "-T", "AT", "-", "", sub_mat, None)
```

Comment:
We should add a test for what happens when `char_complexity_adjustment` is not `None`.